### PR TITLE
Feature migration sf5

### PR DESCRIPTION
**Pull request by stood**

### CODE REVIEW

**Comment by stood** [comment]
contains https://github.com/pomm-project/pomm-bundle/pull/108 and https://github.com/pomm-project/pomm-bundle/pull/107

File: .travis.yml [hunk 1]
```diff
@@ -1,21 +1,22 @@
...
-    - "7.0"
     - "7.1"
     - "7.2"
+    - "7.3"
+    - "7.4"
     - "nightly"
 
 env:
     matrix:
-        - SYMFONY_VERSION=2
```
Comment (chanmix51) [inline .travis.yml:16]: Does this patch break the compatibility with Sf2 applications?

Comment (stood) [reply .travis.yml:16]: The support of sf2 is finish

**Review by rippergun — APPROVED** [review]

**Comment by remigarcia** [comment]
hello, any idea when this will be merged please ?

**Comment by stood** [comment]
@remigarcia it's good
@rippergun  & @chanmix51 thx for you review

**Pull request merged**